modules/Microsoft.Compute/images/.test/common/deploy.test.bicep

```diff
@@ -59,5 +61,16 @@ module testDeployment '../../deploy.bicep' = {
       }
     ]
     zoneResilient: true
+    diskEncryptionSetResourceId: resourceGroupResources.outputs.diskEncryptionSetResourceId
+    extendedLocation: {
+      name: 'customlocal'
+      type: 'EdgeZone'
```

Comment:
@AlexanderSehr this parameter is causing the pipeline to fail for the region we're using

```pwsh
  Status Message: Edge zone 'customlocal' not found. The available edge zones in location 'westeurope' are ''. (Code:UnsupportedEdgeZone)
  
  Status Message: At least one resource deployment operation failed. Please list deployment operations for details. Please see https://aka.ms/DeployOperations for usage details. (Code: DeploymentFailed)
   - {
    "error": {
      "code": "UnsupportedEdgeZone",
      "message": "Edge zone 'customlocal' not found. The available edge zones in location 'westeurope' are ''."
    }
  } (Code:BadRequest)
  
```

Reply:
yeah - it was a try. Removed the test - which didn't stop the pipeline from failing, but I guess it's a step in the right direction :D

Reply:
Done